fixed:#2527 Refactored CSS files in src/screens/UserPortal

gurramkarthiknetha opened the pull request:
<!--
This section can be deleted after reading.

We employ the following branching strategy to simplify the development process and to ensure that only stable code is pushed to the `master` branch:

- `develop`: For unstable code: New features and bug fixes.
- `master`: Where the stable production ready code lies. Only security related bugs.

NOTE!!!

ONLY SUBMIT PRS AGAINST OUR `DEVELOP` BRANCH. THE DEFAULT IS `MAIN`, SO YOU WILL HAVE TO MODIFY THIS BEFORE SUBMITTING YOUR PR FOR REVIEW. PRS MADE AGAINST `MAIN` WILL BE CLOSED.
-->

<!--
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request.
-->

**What kind of change does this PR introduce?**

<!-- E.g. a bugfix, feature, refactoring, etc… -->

**Issue Number:**

Fixes #2527 

**Did you add tests for your changes?**
Yes 

**Snapshots/Videos:**

<!--Add snapshots or videos wherever possible.-->

**If relevant, did you update the documentation?**

<!--Add link to Talawa-Docs.-->

**Summary**

The goal is to convert the CSS file in this subdirectory and all the components related to this screen to use this new design pattern.

<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue for more information. -->

**Does this PR introduce a breaking change?**
NO
<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->

**Other information**

<!--Add extra information about this PR here-->

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes 


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
	- Updated styling for various components across the User Portal, including Campaigns, Chat, Donate, Events, Organizations, People, Pledges, Posts, Settings, UserScreen, and Volunteer sections.
	- Introduced new CSS classes and variables in the global style file for enhanced design consistency and responsiveness.

- **Chores**
	- Removed several unused CSS files to streamline the codebase.
	- Modified import paths for styling to ensure consistent application of CSS across components.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

palisadoes commented on src/screens/UserPortal/Volunteer/VolunteerManagement.module.css:
```diff
@@ -1,138 +0,0 @@
-.greenregbtn {
-  margin: 1rem 0 0;
-  margin-top: 15px;
-  border: 1px solid #e8e5e5;
```
It looks like you have truncated this file to zero bytes. Why wasn't it deleted?

palisadoes commented on src/style/app.module.css:
```diff
@@ -308,13 +197,6 @@
 
-.inputField {
-  margin-top: 10px;
-  margin-bottom: 10px;
```
Why was this removed? Won't it have an impact on other layouts?

gurramkarthiknetha replied:
it's already exists on line 112

palisadoes commented on src/style/app.module.css:
```diff
@@ -223,11 +83,36 @@
 
 .btnsContainer .btnsBlock {
   display: flex;
-  width: max-content;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -1,160 +1,20 @@
 }
 
 .closeButton {
-  margin-bottom: 10px;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -1,160 +1,20 @@
-  margin-bottom: 50px;
-  border-color: #e8e5e5;
-  width: 80%;
-  border-radius: 7px;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -1,160 +1,20 @@
-  margin-right: 40px;
-  color: #737373;
-}
-.cards > h2 {
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -349,21 +256,32 @@
 }
 
-.searchIcon {
-  color: var(--brown-color);
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -389,10 +307,16 @@
+
 .rowBackground {
   background-color: var(--row-background);
-  max-height: 120px;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -401,48 +325,56 @@
-  /* Add error icon for non-color indication */
-  &::before {
-    content: '⚠️';
-    margin-right: 0.5rem;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -401,48 +325,56 @@
-  scrollbar-width: thin;
-  scrollbar-color: var(--bs-gray-400) var(--bs-white);
-  max-height: calc(100vh - 18rem);
-  overflow: auto;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -472,11 +404,6 @@ input[type='radio']:checked + label:hover {
-.datediv {
-  display: flex;
-  flex-direction: row;
-}
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
 }
 
-.pageNotFound {
-  position: relative;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
 
-.pageNotFound h3 {
-  font-family: 'Roboto', sans-serif;
-  font-weight: normal;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
 }
 
-.pageNotFound .brand span {
-  margin-top: 50px;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
 
-.pageNotFound .brand h3 {
-  font-weight: 300;
-  margin: 10px 0 0 0;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
 
-.pageNotFound h1.head {
-  font-size: 250px;
-  font-weight: 900;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
-  top: 50%;
-  width: 50%;
-  height: 1px;
-  background: #fff;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
 }
 
-.pageNotFound h1.head span:after {
-  right: -55%;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
+  }
 
-.pledgeContainer {
-  margin: 0.6rem 0;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
 
-.container {
-  min-height: 100vh;
-}
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
+  }
 
-.pledgeModal {
-  max-width: 80vw;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
-  box-shadow: 0 2px 2px #e8e5e5;
-  padding: 10px 10px;
-  border-radius: 5px;
-  background-color: var(--bs-primary);
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
-.btnsContainerPledge {
-  display: flex;
-  gap: 0.8rem;
-  margin: 2.2rem 0 0.8rem 0;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
 
-.btnsContainerPledge input {
-  outline: 1px solid var(--bs-gray-400);
+.listBox .customTable {
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
-.btnsContainerPledge .inputPledge {
-  flex: 1;
-  min-width: 18rem;
-  position: relative;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
 
-.inputFieldPledge {
-  background-color: white;
-  box-shadow: 0 1px 1px #31bb6b;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
 }
 
-.btnsContainerPledge .inputPledge button {
-  width: 52px;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
-.dropdownPledge {
-  background-color: white;
-  border: 1px solid var(--bs-primary);
-  position: relative;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
 }
 
-.rowBackgroundPledge {
-  background-color: var(--bs-white);
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
 
-.TableImagePledge {
-  object-fit: cover;
-  width: calc(var(--table-image-size) / 2) !important;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
 
-.imageContainerPledge {
-  display: flex;
-  align-items: center;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/style/app.module.css:
```diff
@@ -542,2503 +469,1119 @@ hr {
-  margin: 0.1rem 0.25rem;
-  gap: 0.25rem;
-  padding: 0.25rem 0.45rem;
-  border-radius: 0.35rem;
```
Why was this removed? Won't it have an impact on other layouts?

palisadoes commented on src/screens/UserPortal/Chat/Chat.tsx:
```diff
@@ -105,29 +113,12 @@ export default function chat(): JSX.Element {
-  React.useEffect(() => {
+  useEffect(() => {
     if (chatsListData) {
       setChats(chatsListData.chatsByUserId);
```
Why was this file edited for functionality outside of the CSS? We have an open PR to fix chat.

palisadoes reviewed (changes requested):
1. There are a lot of unexplained changes to the CSS. Please explain.
2. You also changed the logic in the Chat.tsx file. This is out of scope

gurramkarthiknetha commented:
> 1. There are a lot of unexplained changes to the CSS. Please explain.
> 2. You also changed the logic in the Chat.tsx file. This is out of scope

i will keep new pr with in 5hrs with all required changes 

> 1. There are a lot of unexplained changes to the CSS. Please explain.
> 2. You also changed the logic in the Chat.tsx file. This is out of scope

I will submit a new pull request within 5 hours with all the required changes.

gurramkarthiknetha commented:
> 1. There are a lot of unexplained changes to the CSS. Please explain.
> 2. You also changed the logic in the Chat.tsx file. This is out of scope

I will submit a new pull request within 5 hours with all the required changes.

gurramkarthiknetha commented:
<img width="850" alt="image" src="https://github.com/user-attachments/assets/e5527938-ddb4-4099-a81e-65747fd508bb" />
in the file chart.tsx i getting these error when i try to commit

palisadoes commented:
Thanks for submitting your PR, but we must close it.

1. We get notified of every event in the queue, and draft PRs like these, and their updates that require no action can be a distraction. We need our reviewers to focus on updates that are ready. 
1.  Don't be discouraged, we are just trying to keep our workloads to a manageable level.

Please resubmit when you are ready.
